packages/@webex/plugin-meetings/tsconfig.json

```diff
@@ -1,6 +1,41 @@
 {
     "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "module": "es2020",
```

Comment:
I combined rules from the calling plugin and also added some useful type rules from WCME. I'm open to discussing any details about these rules, as I might have missed some crucial points that are necessary for the SDK. I appreciate your effort in asking about this.